### ISSUE TITLE
Possible kills percentage bug

### ISSUE THREAD

**Issue by PoniestPonypony** [bibendovsky/bstone#86]
I did a 100% playthrough of the entire Planet Strike game on difficulty level 2 (skilled agent) in the bstone source port.  I only noticed one minor issue:  In Area 4 (Radiation Containment) I was able to get an "enemy destroyed" percentage of 102%.  Basically I played through the level normally, collecting all treasure and defeating all enemies along the way, and then went to the final room at the bottom of the map where the security cube is located.  There were two Alien Protectors in there; when I killed _one_ of them I got the reward for 100% kills.  Defeating the remaining enemy resulted in the "enemy destroyed" percentage going from 100% to 102%.  I don't know if this glitch happens in the vanilla engine.

**Comment by XxMiltenXx** [comment]
I can confirm this issue, however, I cannot tell you how to reproduce it.
I had it happen on the first Level of Planet Strike where I got 104% kills on Veteran Agent. And since I hadn't collected all treasure yet, I couldn't get the "Total Points" bonus, as it also went above 100% and thusly I also missed out on the "Informants alive" bonus.

I tried to reproduce the bug, so I restarted the level, but this time it counted correctly. So I cannot tell you what's happening there.
